tests/functional/test_issue-249-has/test.feature

```diff
@@ -0,0 +1,18 @@
+# Fixed version of the features provided on issue 249
+# Has: "when it contains" steps are changed to "when it has"
+Feature: Resources should be properly tagged
+  In order to keep track of resource ownership
+  As engineers
+  We'll enforce tagging on all resources
+
+# Check required tag for ALB and CF
+  Scenario: Ensure that waf_policy for ALB/CF tag is defined
+    Given I have aws_alb defined
+    When it has tags
+    Then it must contain waf_policy
+    And its value must match the "^(internal|external|custom)$" regex
```

Comment:
```cucumber
When it has tags
```

will not drill down to the resource. `waf_policy` exists in `tags` parameter, thus the test must be changed like ;

```cucumber
  Scenario: Ensure that waf_policy for ALB/CF tag is defined
    Given I have aws_alb defined
    When it has tags
    Then it must contain tags
    And it must contain waf_policy
    And its value must match the "^(internal|external|custom)$" regex
```